com.unity.render-pipelines.high-definition/Documentation~/whats-new-13.md

```diff
@@ -4,4 +4,13 @@ This page contains an overview of new features, improvements, and issues resolve
 
 ## Added
 
+### HDR Output Support
+
+HDRP 13.0 introduces support to output to HDR Displays, supporting both  HDR10 and scRGB standards.
+
+When outputting to an HDR-capable device, if the project enabled HDR Output, HDRP will make use of the higher brightness contrast and wider color gamut capabilities of the display.
+The user can also chose for various options on how to customize the tonemapping for the HDR case and few options are provided to allow the game to adapt to various output displays and calibrate basing on device metadata or user preferences.
+
+For more information consult the [HDR Output](HDR-Output.md) documentation
```

Comment:
Thank you so much for including the upgrade guide and what's new updates in your PR! : D

Proposal:

"HDRP 13.0 introduces support for HDR display output, including both the HDR10 and scRGB standards.

As a result, HDRP is now able to take advantage of the higher brightness contrast and wider color gamut capabilities of HDR displays.

This functionality includes a variety of customization options for adapting content for a variety of displays based on device metadata or user preferences. 

For more information consult the [HDR Output](HDR-Output.md) documentation."

Reply:
There are a few more items above I don't see a response on, but I'm not going to @ you in all of them, just in this last one (: @FrancescoC-unity

Reply:
Realized that some of my comments were Pending as review comments. Am correcting this now and also at-ing you on each thread so you can find it more easily.